align_system/prompt_engineering/outlines_prompts.py

```diff
@@ -940,3 +940,43 @@ def scenario_description_hybrid_regression(scenario_state):
     """
     {{ scenario_state.unstructured.rstrip() }} {% for character in scenario_state.characters %}{{  character.name }} - {{ character.unstructured.rstrip()}} {% endfor %}
     """
+
+
+def opinion_qa_system_prompt(kdma, value):
```

Comment:
@barry-ravichandran this is an import function for your review, this is how we're building the aligned system prompts.